Automattic/simplenote-ios
# Fixing Update-Translations Script

### Fix
In this PR we're replicating WPiOS's [Update Translations](https://github.com/wordpress-mobile/WordPress-iOS/blob/develop/Scripts/update-translations.rb) script.

Instead of downloading the JSON + Strings translation, we snap everything in a single shot (which is what we do in WordPress).

The current mechanism has a couple issues, and it's producing malformed strings / comments. [More here!](https://github.com/Automattic/simplenote-ios/pull/544).

Gentlemen, may I bug you with this PR?
cc @loremattei or @jkmassel

Thanks in advance!!

 
### Test
1. Run `./Scripts/update-translations.rb`
2. Verify Simplenote builds correctly
3. (OR) simply verify that the CI is happy

**Note:** Since I was in the neighborhood, I've also squeezed in the fixed `.strings` files!!

### Review
These changes do not require release notes.
